mint.json

```diff
@@ -57,12 +57,17 @@
 		{
 			"name": "Changelog",
 			"icon": "code-commit",
-			"url": "https://plain.com/changelog"
+			"url": "https://plain.com/changelog&utm_source=docs"
```

Comment:
Can we update all URLs to include the `?` to separate the path and the query args?